roles/zookeeper/files/zookeeper-wait-for-listen.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+[[ -n "$1" ]] && host=$1 || host=$HOSTNAME
```

Comment:
Alternate and a bit simpler:

```
host=${1:-"$HOSTNAME"}
```

It will assign input $1 if provided, but the default value is $HOSTNAME when $1 provided.